native-auth/config.md

```diff
@@ -0,0 +1,11 @@
+## NativeAuthClientConfig
+
+```
+dto NativeAuthClientConfig:
+    origin?: string;
+    api_url?: string;
+    expiry_time_in_seconds?: int;
+    block_hash_shard?: int;
+    gateway_url?: string;
```

Comment:
Proxy URL maybe, since we say "ProxyNetworkProvider"? (debatable though).

Reply:
Will rename it in the next PR.